src/interactive/mod.rs

```diff
@@ -253,7 +253,9 @@ impl App {
                         .change_selected(&visible, CursorMove::Absolute(usize::MAX));
                     Refresh::Update
                 }
-                KeyCode::PageUp => {
+                KeyCode::PageUp | KeyCode::Char('u')
+                    if key.modifiers.contains(KeyModifiers::CONTROL) =>
```

Comment:
Is there an easy way to pass through multiple cases? Like in a traditional switch block having multiple cases that go to the same body

Reply:
I don't think so. It would be less strict and its easy to create bugs that way so it probably wasn't implemented in Rust? I would assume it would be shown in the [Rust book match page](https://doc.rust-lang.org/book/ch06-02-match.html) but it isn't so I assume it doesn't exist.

The code for the case is already fairly simple and just calling other code already made to reduce duplicate code so duplicating it for the vim keys doesn't seem that bad to me.

Reply:
@lukepighetti do you want to change that? If you don't have the time for that I can take a look on that.

Reply:
I tried to extract it into a function but was having trouble (new to Rust). So I duplicated it. Happy to make any changes.

Reply:
I already created the change_selected function there to simplify the different cursor keys. So I think this is the best way to do it for now.